Sedah/Assets/spawnPlayerChars.cs

```diff
@@ -0,0 +1,14 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Mirror;
+public class spawnPlayerChars : NetworkBehaviour
```

Comment:
nit: Class and file name should be capitalized

Reply:
I need to delete this script. Obsolete.